nodes/textures/iorpreset.py

```diff
@@ -0,0 +1,71 @@
+"""A BlendLuxCore node to provide index of refraction preset values for
+    LuxCoreRender in Blender"""
+# <pep8 compliant>
+import bpy
+from bpy.props import FloatProperty, StringProperty
+from .. import LuxCoreNodeTexture
+from ...operators import ior_presets
+
+
+class LuxCoreNodeIORPreset(LuxCoreNodeTexture):
```

Comment:
All texture nodes are prefixed with "LuxCoreNodeTex".
So the name should be "LuxCoreNodeTexIORPreset".